create.html

```diff
@@ -37,7 +37,6 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
```

Comment:
What part of the code base is using this create.html file? If we need to use it somewhere, it's missing dependencies.